ShowOrchard.Theme/Views/Layout.liquid

```diff
@@ -21,7 +21,7 @@
 
     {% render_section "HeadMeta", required: false %}
 
-    {% shape "GoogleTag", GoogleTagPropertyId: "G-C8EEHMLQLP", CookieDomain: "auto" %}<!-- #spell-check-ignore-line -->
+    {% google_tag property_id: "G-C8EEHMLQLP", cookie_domain: "auto" %}<!-- #spell-check-ignore-line -->
```

Comment:
This isn't rendering. I changed the `if` to just `true` in _GoogleTag.cshtml_ to enable it locally, and it does run, but in the end, the `<script>` is not there in the layout.

Reply:
I don't get it, did this shape ever work? The problem is not with the Liquid tag but the the fact that the GoogleTag.cshtml had `<script at="Head">`. This shape invocation is [below](https://github.com/Lombiq/Show-Orchard-Theme/blob/519948b0d4bde553651acdf5bd3ad4026c04d0d2/ShowOrchard.Theme/Views/Layout.liquid#L18) the `{% resources type: "HeadScript" %}`, so at this point it's already too late to add scripts to the resource manager.
I just [removed](https://github.com/Lombiq/Helpful-Extensions/pull/176/commits/a3a68e6ff2e22a1ef7e0665f06315142a5c4e7e2) the `at="Head"` and it works now.

Reply:
I don't know if this ever worked, but indeed it's not working currently on the prod site. Thank you!